spec/pg/connection_spec.rb

```diff
@@ -821,6 +821,33 @@ def interrupt_thread(exc=nil)
 			end
 		end
 
+		it "commits even if the block includes an early break/return" do
+			# abort the per-example transaction so we can test our own
+			@conn.exec( 'ROLLBACK' )
+
+			res = nil
+			@conn.exec( "CREATE TABLE pie ( flavor TEXT )" )
+
+			begin
+				res = @conn.transaction do
```

Comment:
This assignment and the one above are unnecessary. Could you please remove them.

Reply:
Good catch thanks, removed in [48bc9b2](https://github.com/ged/ruby-pg/pull/463/commits/48bc9b26f5bb76d53cb2abb7c7d55da1db0c58ba)